test/sample-test/sample_test_launcher.py

```diff
@@ -71,58 +85,113 @@ def check_notebook_result(self):
 
```

Comment:
This is also not used any more, correct?  maybe remove it?

Reply:
Done.